.cci.jenkinsfile

```diff
@@ -0,0 +1,112 @@
+@Library('github.com/coreos/coreos-ci-lib@master') _
+
+stage("Build") {
+parallel rpms: {
+  coreos.pod(image: 'registry.fedoraproject.org/fedora:30', privileged: true) {
+      checkout scm
+      sh """
+        set -euo pipefail
+        ci/installdeps.sh
+        git submodule update --init
+
+        # We lose sanitizers (all the *san) here by building straight to RPMs, but we can
+        # restore those through a build opt later on. Being able to stash RPMs directly is
+        # super nice (and archiving later on will make it easy for anyone to download
+        # binaries from PRs in the future) and meshes well with the following stages.
+        export PATH="/root/.cargo/bin:\$PATH"
+        cargo install cbindgen
+        cbindgen -c rust/cbindgen.toml -o rpmostree-rust.h rust
+
+        cd packaging
+        make -f Makefile.dist-packaging rpm
+      """
+      stash includes: 'packaging/**/*.rpm', name: 'rpms'
```

Comment:
This is an interesting approach.  Worth contrasting with e.g. https://github.com/projectatomic/rpm-ostree/pull/1893 where what I was trying to do was get closer to a model where we build a derived container.  You could imagine e.g. that we generate a `FROM coreos-assembler` container and then test *that* potentially multiple ways.

That said, we need RPMs for composes, so I'm OK with this approach too.